indexer.go

```diff
@@ -1,31 +1,77 @@
 package indexer
 
+import (
+	"github.com/pokt-foundation/pocket-go/provider"
+)
+
 // Provider interface of needed provider functions
 type Provider interface {
 	GetBlockHeight() (int, error)
-	// TODO: add missing functions
+	GetBlock(blockHeight int) (*provider.GetBlockOutput, error)
+	GetBlockTransactions(blockHeight int) (*provider.GetBlockTransactionsOutput, error)
 }
 
-// Writer interface of write function
-type Writer interface {
-	// TODO: add missing functions
+// Persistance layer interface (database, in-memory, etc.)
+type Persistance interface {
```

Comment:
```suggestion
type Persistence interface {
```

Reply:
lol didn't realize the typo even tho i looked at it for hours